### PR TITLE
Allow build temporary Dockerfile in configured custom_dir

### PR DESCRIPTION
**Pull request by Val**

### CODE REVIEW

**Comment by jeffreycoe** [comment]
Hi @Val - Could you provide more context into what this PR is attempting to fix? Thanks.

**Comment by Val** [comment]
Hi @jeffreycoe 

This PR fix possible limited rights on `Dir.pwd` whem using **dockerd** with `--userns-remap` aka [user namespace isolation](https://docs.docker.com/engine/security/userns-remap/).

**Comment by Val** [comment]
@damacus can you please review :pray:

File: README.md [hunk 1]
```diff
@@ -583,6 +583,16 @@ Examples:
     net: br3
 ```
 
+### build_tempdir
+
+Relative (to `build_context`) temporary directory path for builded Dockerfile.
```
Comment (damacus) [inline README.md:588]:
```suggestion
Relative (to `build_context`) temporary directory path for built Dockerfile.
```

**Review by damacus — APPROVED** [review]

**Comment by damacus** [comment]
Ace! Thanks very much for this.

**Comment by Val** [comment]
Thank you ! :pray:

**Pull request merged**